oak-parent/pom.xml

```diff
@@ -61,7 +61,7 @@
     <slf4j.version>1.7.32</slf4j.version> <!-- sync with logback version -->
     <logback.version>1.2.10</logback.version>
     <h2.version>2.0.206</h2.version>
-    <tika.version>1.24.1</tika.version>
+    <tika.version>1.26</tika.version>
```

Comment:
I believe we should upgrade to the latest 1.x release (https://mvnrepository.com/artifact/org.apache.tika/tika-core/1.28.5)

Reply:
+1